app/game/memoryGame/script.js

```diff
@@ -0,0 +1,83 @@
+const canvas0 = document.getElementById("canvas0"),
+canvas1 = document.getElementById("canvas1"),
+canvas2 = document.getElementById("canvas2"),
+canvas3 = document.getElementById("canvas3"),
+canvas4 = document.getElementById("canvas4"),
+canvas5 = document.getElementById("canvas5"),
+canvas6 = document.getElementById("canvas6"),
+canvas7 = document.getElementById("canvas7"),
+canvas8 = document.getElementById("canvas8");
+
+let pattern = [0, 0, 0, 0, 0];
+for (let i = 0; i < 5; i++) {
+    pattern[i] = Math.floor(Math.random() * 9);
+    renderPattern();
+}
+console.log(pattern);
+
+function renderPattern() {
+    for (let i = 0; i < pattern.length; i++) {
+        setTimeout(() => {
+            const canvas = document.getElementById(`canvas${pattern[i]}`)
+            canvas.style.background = "lightgreen";
+            setTimeout(() => {
+                canvas.style.background = "lightgray";
+            }, 1000);
+        }, 1200 * i);
```

Comment:
Let's pull these values into named constants at the top of the file.
Something like `blinkDuration` and `timeBetweenBlinks`.
I also think we should lower these values to be pretty quick. The whole minigame should take like 5 seconds ideally